src/content/en/showcase/2017/playcanvas.md

```diff
@@ -0,0 +1,201 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/playcanvas/vr-lab.png #}
+{# wf_featured_snippet: The PlayCanvas WebVR Lab is a living project from the PlayCanvas team. It represents the team's research into developing scalable and responsive WebVR applications for all devices. It elegantly scales from a Google Cardboard or a Daydream View headset to a desktop VR device. The project is continually updated with new experiments that implement core interactions like teleportation, grabbing, manipulating objects, user interface and controllers. #}
+{# wf_tags: webvr,casestudy #}
+
+# Core Interactions in the WebVR Lab {: .page-title }
+
+*By Dave Evans, Chief Technology Officer, PlayCanvas*
+
+<img src="/web/showcase/2017/images/playcanvas/vr-lab.png" class="attempt-right">
+
+<a class="button button-primary" href="http://webvr.playcanvas.com">
+  View the Lab
+</a>
+
+Website designers are familiar with the idea of responsive design, building a
+website to work across different devices, such as mobile phones, tablets and
+computers and different screen resolutions and pixel ratios. WebVR introduces
+new challenges to the idea of responsive design.
+
+We can categorize a WebVR experience by two metrics - the level of immersion and
+the level of input. These levels are distinguished by "degrees of freedom"
+(DOF). 6 DOF refers to tracking both position and orientation, 3 DOF is only
+tracking orientation and 0 DOF tracks neither position nor orientation.
+
+### Immersion Level
+
+- No VR - monitor or mobile display.
```

Comment:
Optional: wrap each of the key terms (No VR, Stationary VR, etc) in bold or something so they stand out more. Same for the input level.